src/Databases/SQLite/fetchSQLiteTableStructure.cpp

```diff
@@ -31,17 +31,10 @@ static DataTypePtr convertSQLiteDataType(String type)
     DataTypePtr res;
     type = Poco::toLower(type);
 
-    if (type == "tinyint")
-        res = std::make_shared<DataTypeInt8>();
-    else if (type == "smallint")
-        res = std::make_shared<DataTypeInt16>();
-    else if ((type.starts_with("int") && type != "int8") || type == "mediumint")
-        res = std::make_shared<DataTypeInt32>();
-    else if (type == "bigint" || type == "int8")
```

Comment:
They are just ["hints"](https://www.sqlite.org/datatype3.html) in sqlite. In recent versions one could use `STRICT` [keyword](https://www.sqlite.org/stricttables.html) - but then it's an `int64_t` again...
It might be a bit unexpected an confusing - not sure what's the best way for the Clickhouse world.